package.json

```diff
@@ -46,6 +45,7 @@
     "react-diff-viewer": "^3.1.1",
     "react-dom": "^17.0.1",
     "react-feather": "^2.0.9",
+    "sass": "^1.32.8",
```

Comment:
`node-sass` was causing problems with css modules. It's [recommended](https://github.com/sass/node-sass) to move to `dart-sass` (`sass`) anyways.